pulp.spec

```diff
@@ -476,6 +476,8 @@ KEY_PATH="$KEY_DIR/rsa.key"
 KEY_PATH_PUB="$KEY_DIR/rsa_pub.key"
 if [ ! -f $KEY_PATH ]
 then
+  # Ensure the key generated is only readable by the owner.
+  umask 077
```

Comment:
Cool. I definitely don't even know what a scriptlet is, so I'm happy with however you want to deal with this. Thanks!